corehq/apps/sso/migrations/0007_alter_identityprovider_idp_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.24 on 2024-03-05 16:06
```

Comment:
@jingcheng16 can you cherry pick the commit that actually alters the identity provider and create a new PR from that?

Reply:
FYI matt has a PR open for another migration also coming after `0006...` cc @mjriley https://github.com/dimagi/commcare-hq/pull/34248#event-12067141878